model/scratch/dffml_model_scratch/logisticregression.py

```diff
@@ -82,14 +141,16 @@ def best_fit_line(self):
         learning_rate = 0.01
         w = 0.01

```

Comment:
Are `learning_rate`, `w`, `b`, and `1500` something the user might want to configure? It seems odd that they are hardcoded. What do they mean? If they are something the user might want to tweak, then they should go in `LogisticRegressionConfig`.